.circleci/scripts/det_deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+det deploy $@
```

Comment:
```suggestion
det deploy "$@"
```
You need to quote this variable in order to keep parameters with spaces together as-passed.